### PR TITLE
added "epochms" and "epoch" masks to DateTimeFormat

### PR DESCRIPTION
**Pull request by cfmitrah**
https://luceeserver.atlassian.net/browse/LDEV-2439

### CODE REVIEW

**Comment by zspitzer** [comment]
can add you add epochms to the documentation?

**Comment by cfmitrah** [comment]
@zspitzer done in (https://github.com/lucee/lucee-docs/pull/1097)

**Comment by zspitzer** [comment]
I meant in the .fld file in this PR :)

File: core/src/main/java/resource/fld/core-cfml.fld [hunk 1]
```diff
@@ -451,6 +451,7 @@ The following masks can be used to format the full date and time and may not be
 - long: medium followed by three-letter time zone; i.e. "mmmm d, yyyy h:nn:ss tt zzz"
 - full: equivalent to "dddd, mmmm d, yyyy h:nn:ss tt zz"
 - ISO8601/ISO: equivalent to "yyyy-mm-dd'T'HH:nn:ssXXX"
+- epoch: Total seconds of a given date (Example:1567517664)
```
Comment (zspitzer) [inline core/src/main/java/resource/fld/core-cfml.fld:454]: epochms is missing

**Review by zspitzer — CHANGES_REQUESTED** [review]

File: core/src/main/java/resource/fld/core-cfml.fld (hunk 1 above)
Comment (cfmitrah) [reply core/src/main/java/resource/fld/core-cfml.fld:454]: Added that too.

**Pull request merged**